mypy/newsemanal/typeanal.py

```diff
@@ -285,6 +285,12 @@ def try_analyze_special_unbound_type(self, t: UnboundType, fullname: str) -> Opt
                       " in a variable annotation", t)
             return AnyType(TypeOfAny.from_error)
         elif fullname == 'typing.Tuple':
+            # Tuple is special because it is involved in builtin import cycle
+            # and may be not ready when used.
+            sym = self.api.lookup_fully_qualified_or_none('builtins.tuple')
+            if not sym or isinstance(sym.node, PlaceholderNode):
```

Comment:
What happens if we don't use the correct fixture that defined `tuple`? Will this still generate a reasonable error message?

Reply:
I fixed this to report a reasonable error, and added a test.